manual/manual/cmds/native.etex

```diff
@@ -566,6 +558,25 @@ The Sparc code generator supports the following additional options:
 The default is to generate code for SPARC version 7, which runs on all
 SPARC processors.
 
+\paragraph{contextual control of command-line options}
+
+The compiler command line can be modified ``from the outside''
+with the following mechanisms. These are experimental
+and subject to change, and they are not intended for normal use.
+
+\begin{options}
+\item["OCAMLPARAM" \rm(environment variable)]
+Arguments that will be inserted before or after the arguments from the
+command line.
+\item["ocaml_compiler_internal_params" \rm(file in the stdlib directory)]
+A mapping of file names to lists of arguments that
+will be added to the command line (and "OCAMLPARAM") arguments.
+\item["OCAML_FLEXLINK" \rm(environment variable)]
+Alternative executable to use on native
+Windows for "flexlink" instead of the
+configured value. Primarily used for bootstrapping.
+\end{options}
+
 \section{Common errors}
```

Comment:
Concerning the duplication between `comp.etex` and `native.etex`, I had proposed to merge these files together (with `top.etex`) and handle the differences with tex macros in #491 . If there are some interests in this solution, I could rebase this PR; or create a version merging only `comp.etex` and `native.etex` − which would decrease the complexity of the resulting tex file.